jekyll/_cci2/install-machine-runner-3-on-osx.adoc

```diff
@@ -0,0 +1,179 @@
+---
+contentTags:
+  platform:
+  - Cloud
+---
+= Install machine runner 3.0 on macOS - Open preview
+:page-layout: classic-docs
+:page-liquid:
+:page-description: Instructions on how to install CircleCI's self-hosted machine runner (3.0) on macOS.
+:icons: font
+:experimental:
+:machine:
+:macOS:
+
+NOTE: Machine runner 3.0 is currently in open preview
+
+This page describes how to install CircleCI's machine runner 3.0 on macOS.
+
+[#prerequisites]
+== Prerequisites
+
+{% include snippets/runner/machine-runner-prereq.adoc %}
+
+[#self-hosted-runner-terms-agreement]
+== Self-hosted runner terms agreement
+
+{% include snippets/runner/terms.adoc %}
+
+[#create-namespace-and-resource-class]
+== 1. Create namespace and resource class
+
+{% include snippets/runner/install-with-cli-steps.adoc %}
+
+[#install-circleci-runner]
+== 2. Install CircleCI runner on macOS
+
+
+The easiest way to install runner on macOS is to use link:http://brew.sh/[Homebrew].
+
+On the target macOS machine with homebrew installed, run the following command to add the CircleCI repository:
+
+```shell
+brew tap circleci-public/circleci
+```
+
+Next, install the circleci-runner package by running the following command:
+
+
+Note: You may see an expected notification indicating a background item for Circle Internet Services Inc. has been added.
+
+```shell
+brew install circleci-runner
+```
+
+Next, open the runner config.yaml file with the text editor of your choice, and modify the `[[RUNNER_NAME]]` and `[[RESOURCE_CLASS_TOKEN]]` template values.
+
+```shell
+nano $HOME/Library/Preferences/com.circleci.runner/config.yaml
+```
+
+```
+runner:
+  name: "my-osx-runner"
+  working_directory: "/Users/yourusername/Library/com.circleci.runner/workdir"
```

Comment:
I think we can safely swap `yourusername` for $USER to make it more pastable

Reply:
good catch.

Reply:
```suggestion
  working_directory: "/Users/$USER/Library/com.circleci.runner/workdir"
```